VoodooI2CELAN/VoodooI2CELANTouchpadDriver.hpp

```diff
@@ -8,8 +8,10 @@
 
 #ifndef VOODOOI2C_ELAN_TOUCHPAD_DRIVER_HPP
 #define VOODOOI2C_ELAN_TOUCHPAD_DRIVER_HPP
+#define INTERRUPT_SIMULATOR_TIMEOUT 5
```

Comment:
Don't put any definition next to the include guard. Put it after the `ELAN_NAME` definition instead, for example :)

Reply:
Done